mm2src/mm2_main/src/lp_swap/taker_swap_v2.rs

```diff
@@ -1772,6 +1816,113 @@ impl<MakerCoin: MmCoin + MakerCoinSwapOpsV2, TakerCoin: MmCoin + TakerCoinSwapOp
     }
 }
 
+struct TakerPaymentSentAndPreimageSendingSkipped<MakerCoin: ParseCoinAssocTypes, TakerCoin: ParseCoinAssocTypes> {
```

Comment:
ooh, there is a new struct but you didn't use it.

Reply:
It is used in next states, i forgot to change struct name in recreate_machine